IaC/PnP/monitoring/aks/online/aks_secure_baseline/cluster-baseline-settings/ingress-network-policy.yaml

```diff
@@ -0,0 +1,16 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-only-ingress-to-workload
+  namespace: a0008
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: aspnetapp
```

Comment:
This is worry-some that it falls under the `cluster-baseline-settings` but is clearly tied to the specific `aspnetapp` workload. We should move this and make sure we have similar NetworkPolicies for all applications

Reply:
rather than refactoring a bunch of how CAF has scripted this, we will move that as a concern to the Deploy team rather than doing a refactor as part of getting something deployable for monitoring.